core/src/main/scala/chisel3/Annotation.scala

```diff
@@ -81,7 +77,7 @@ object annotate {
   *
   * @note Calling this on [[Data]] creates an annotation that Chisel emits to a separate annotations
   * file. This file must be passed to FIRRTL independently of the `.fir` file. The execute methods
-  * in [[chisel3.Driver]] will pass the annotations to FIRRTL automatically.
+  * in `chisel3.Driver` will pass the annotations to FIRRTL automatically.
   */
```

Comment:
We should just delete references to `chisel3.Driver` (or perhaps replace them at times with a reference to `ChiselStage`).

In this case, we should just delete this whole `@note` since this API cannot be called on `Data`, it only applies to modules.